node-net-manager/handlers/deployment.go

```diff
@@ -83,7 +84,13 @@ func deploymentHandler(requestStruct *ContainerDeployTask) (net.IP, error) {
 	}
 
 	//attach network to the container
-	addr, err := requestStruct.Env.AttachNetworkToContainer(requestStruct.Pid, requestStruct.ServiceName, requestStruct.Instancenumber, requestStruct.PortMappings)
+	var addr net.IP
+	var err error
+	if requestStruct.IsUnikernel {
```

Comment:
It would be nice to switch across the virtualization technologies available. 
Sort of 

switch:
case requestStruct.virtualization == virtualization.CONTAINER
...
case requestStruct.virtualization == virtualization.UNIKERNEL